hbase-backup/src/test/java/org/apache/hadoop/hbase/backup/TestBackupMerge.java

```diff
@@ -124,4 +128,44 @@ public void TestIncBackupMergeRestore() throws Exception {
     admin.close();
     conn.close();
   }
+
+  @Test
+  public void TestIncBackupMergeRestoreSeparateFs() throws Exception {
+
+    // prepare BACKUP_ROOT_DIR on a different filesystem from HBase
+    File tempDir = new File(FileUtils.getTempDirectory(), UUID.randomUUID().toString());
+    tempDir.deleteOnExit();
+    BACKUP_ROOT_DIR = tempDir.toURI().toString();
+    System.out.println(BACKUP_ROOT_DIR);
```

Comment:
It's not totally unheard of to print logs like this, especially in our test files, but maybe we should just use the logger?

Reply:
@rmdmattingly  I've removed the offending debug statement